### PR TITLE
Ria 2239 judge end appeal

### PR DESCRIPTION
**Pull request by burale**
### JIRA link (if applicable) ###

[RIA-2239](https://tools.hmcts.net/jira/browse/RIA-2239)

### Change description ###
Changed end-appeal feature to add judge user 

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: e2e/features/case-progression-judge.feature [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+
+    And I switch to be a `Case Officer`
+    And I start decision and reasons
+
+    When I switch to be a `Judge`
+    And I click the `Overview` tab
+
+    Then I should only see the `caseOfficer_decision` case progress image
+
+    # implement within RIA-1280
```
Comment (raghera) [inline e2e/features/case-progression-judge.feature]: can we get rid of these comments?

**Comment by raghera** [comment]
Looks good.  QA may consider if enough coverage.

File: e2e/features/end-appeal.feature [hunk 2]
```diff
@@ -53,9 +53,56 @@ Feature: End appeal
...
 
-  @regression @end-appeal-after-listing @RIA-1677 @RIA-1678 @RIA-1486
-  Scenario Outline: End appeal after listing
+  @regression @end-appeal @RIA-823 @RIA-1677 @RIA-1678 @RIA-1766 @RIA-1486 @RIA-2239
+  Scenario Outline: End appeal after submit appeal judge
+    
+    And I switch to be a `Judge`
+    Then I end the appeal
+
+    And I click the `Documents` tab
```
Comment (bradsorour) [inline e2e/features/end-appeal.feature]: Should possibly use 'When I click...'

File: e2e/features/case-progression-judge.feature (hunk 1 above)
Comment (burale) [reply e2e/features/case-progression-judge.feature]: Done

**Pull request merged**